daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Effects.scala

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.transaction
+
+import com.daml.lf.value.Value
+import com.daml.lf.value.Value.ContractId
+
+import scala.collection.immutable.{TreeMap, TreeSet}
+
+/** The effects of a transaction, that is:
```

Comment:
Are the comments in the original `InputsAndEffects.Effects` regarding semantics of consumed/create contracts captured somewhere else in the code?